src/main/java/org/folio/edge/config/SecurityConfig.java

```diff
@@ -24,23 +25,24 @@
 @RequiredArgsConstructor
 @Configuration
 @Log4j2
-public class SecurityConfig extends WebSecurityConfigurerAdapter {
+public class SecurityConfig {
 
```

Comment:
According to Spring Doc in Spring Boot 3.0.0 (from Spring Framework 5.8) `WebSecurityConfigurerAdapter `was deprecated and  `SecurityFilterChain `was introduced.
So `WebSecurityConfigureAdapter `was removed and `authorizeRequests() `deprecated method changed to `authorizeHttpRequests()`

- https://docs.spring.io/spring-security/reference/5.8/migration/servlet/config.html#_stop_using_websecurityconfigureradapter (SecurityFilerChain)
- https://docs.spring.io/spring-security/reference/5.8/migration/servlet/config.html#use-new-requestmatchers (anyMatchers to requestMatchers)
- https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter